packages/example-app/pages/index.tsx

```diff
@@ -1,7 +1,13 @@
+import { inputOf, outputOf } from "@next-fetch/swr";
 import { useAllPeople } from "./api/people.swr";
 
 export default function Home(props: { runtime: string }) {
-  const result = useAllPeople({ name: "gal" });
+  const data: inputOf<typeof useAllPeople> = {
+    name: "gal",
+  };
```

Comment:
Can we move the `typeof` to the type handler so the usage can be `inputOf<useAllPeople>` @Schniz

Reply:
TypeScript doesn’t support it afaik. useAllPeople is a value, not a type